src/plotrecipes/histograms.jl

```diff
@@ -0,0 +1,9 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+@recipe function f(hist::EmpiricalHistogram)
+  seriestype --> :step
+  seriescolor --> :black
+  hist.hist
```

Comment:
I have no idea why this is not working as expected. Can you maybe try to introduce a series explicitly with:

```julia
@series begin
  seriestype --> :step
  seriescolor --> :black
  hist.hist
end
```

Reply:
Tried this but still not working as expected